docs/userguide/torch_forecasting_models.md

```diff
@@ -461,6 +461,122 @@ model.fit(...)
 
 *Note* : The callback will give one more element in the `loss_logger.val_loss` as the model trainer performs a validation sanity check before the training begins.
 
+#### Example with MLflow Autologging
```

Comment:
Can you please also add an entry in the table of content at the top of the file/page?

Reply:
Done via commit https://github.com/unit8co/darts/pull/2092/commits/ce635bc838f2bc187a9b1cff18f029df45bd7265